test/integration/pack/test_pack.py

```diff
@@ -1014,6 +1015,8 @@ def test_pack_tgz_links_to_binaries(tt_cmd, tmp_path):
 
     tt_is_link = os.path.islink(os.path.join(extract_path, "bin", "tt"))
     tnt_is_link = os.path.islink(os.path.join(extract_path, "bin", "tarantool"))
+    tnt_is_link = os.path.islink(os.path.join(extract_path, "bin", "tcm"))
```

Comment:
```suggestion
    tcm_is_link = os.path.islink(os.path.join(extract_path, "bin", "tcm"))
```
Is there misprint here?

Reply:
Done